src/pages-and-resources/discussions/app-list/messages.js

```diff
@@ -79,6 +79,11 @@ const messages = defineMessages({
     defaultMessage: 'WCAG 2.1 Support',
     description: 'The name of a discussions feature.',
   },
+  supportedFeatureList: {
```

Comment:
I am not certain about the naming conventions here, I would defer to @davidjoy here. 
Personally, I'd move it next to `supportedFeatures` on the top and for the name, I'd name it `supportedFeaturesMobile`, but I could be wrong :)

Reply:
changed it according to my understanding